tests/test_lab_data_generate.py

```diff
@@ -0,0 +1,61 @@
+# Standard
+from unittest import mock
+
+# Third Party
+from click.testing import CliRunner
+
+# First Party
+from instructlab import lab
+from instructlab.configuration import DEFAULTS
+
+
+@mock.patch("openai.OpenAI")
+@mock.patch("httpx.Client")
+@mock.patch("instructlab.model.backends.backends.select_backend")
+@mock.patch("instructlab.sdg.generate_data.generate_data")
+def test_generate_cert_params(
```

Comment:
I don't see a case that tests this condition `"--endpoint-ca-cert and --enable-insecure are mutually exclusive"`, am I missing something?

Reply:
I have added a unit test for that case.